src/main/java/com/team4/testingsystem/services/impl/TestsServiceImpl.java

```diff
@@ -42,7 +41,7 @@ public Test getById(long id) {
     public long createForUser(long userId) {
 
 
-        User user = usersRepository.findById(userId).orElseThrow(UserNotFoundException::new);
+        User user = usersService.getUserById(userId);
```

Comment:
what will happen if a user with the id is not found?

Reply:
`.getUserById(userId)` throws `UserNotFoundException`

Reply:
yeah, it is a service, sure